### PR TITLE
1.0.0-rc2

### PR DESCRIPTION
**Pull request by rtwfroody**
This includes all feedback from the public review period, as well as a bugfix for #985 which was identified after the public review period ended.

This PR cannot be merged until ARC approves it.

### CODE REVIEW

File: xml/hwbp_registers.xml [hunk 1]
```diff
@@ -1515,8 +1521,9 @@ same project unless stated otherwise.
         The reset values listed here apply to every underlying trigger.
 
         This register is accessible as {csr-tdata3} when {tdata1-type} is 2, 3, 4,
-        5, or 6 and XLEN=64. The fields are defined
-        above, in {csr-textra32}.
+        5, or 6 and XLEN=64. The function of the fields are defined
+        above, in {csr-textra32}. This register retains its value when XLEN
+        changes. When XLEN=32 some of the bits can be accessed through {csr-text32}.
```
Comment (ved-rivos) [inline xml/hwbp_registers.xml:1526]:
```suggestion
        changes. When XLEN=32 some of the bits can be accessed through {csr-textra32}.
```

Comment (rtwfroody) [reply xml/hwbp_registers.xml]: Good catch. Fixed. Thank you.

**Comment by rtwfroody** [comment]
Per https://lists.riscv.org/g/tech-chairs/message/1898 this was approved by ARC.

@pdonahue-ventana can you approve this PR so I can merge it?

**Review by pdonahue-ventana — APPROVED** [review]
OK

**Pull request merged**